ui/send_page.go

```diff
@@ -755,8 +780,26 @@ func (pg *SendPage) watchForBroadcastResult() {
 }
 
 func (pg *SendPage) changeEvt(evt widget.EditorEvent) {
+	// this adjusts the width of the amount editor per text input.
```

Comment:
This would be better as the comment to the method. The comment should also include what values specified in the switch statement do. Why those values?